Source/DafnyRuntime/DafnyRuntime.go

```diff
@@ -459,7 +508,16 @@ func SeqOfChars(values ...Char) Seq {
   return Seq{arr, true}
 }
 
+func SeqOfCodePoints(values ...CodePoint) Seq {
```

Comment:
Isn't this the same as `SeqOf` and `SeqOfChars` above?

Reply:
Yup, as per the other comment I nuked this